sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -181,6 +181,8 @@ case class CreateViewCommand(
    * Permanent views are not allowed to reference temp objects, including temp function and views
    */
   private def verifyTemporaryObjectsNotExists(sparkSession: SparkSession): Unit = {
+    import sparkSession.sessionState.analyzer._
```

Comment:
Nit: Since only `AsTableIdentifier` is used and analyzer has a lot of child classes, I think it would be better to import just `AsTableIdentifier` and not `_`.